packages/clay-nav/docs/index.js

```diff
@@ -123,4 +124,94 @@ const VerticalNavigation = () => {
 	);
 };
 
-export {Navigation, VerticalNavigation};
+const verticalNavigationCustomItemImportsCode = `import {VerticalNav} from '@clayui/nav';
+import ClayIcon from '@clayui/icon';
+`;
+
+const VerticalNavigationCustomItemCode = `const Component = () => {
+	const noIcons = [1, 5];
+
+	return (
+		<ClayVerticalNav
+			active={6}
+			defaultExpandedKeys={new Set([5])}
+			items={[
+				{
+					id: 1,
```

Comment:
Hi @pat270 could you change the types of `id` to `string`? I added some explanation about this in PR #5602